tmt/package_managers/bootc.py

```diff
@@ -99,7 +101,12 @@ def _get_current_bootc_image(self) -> str:
             raise tmt.utils.PrepareError(f"Failed to extract bootc image info: missing {error}")
 
     def _get_base_containerfile_directives(self) -> list[str]:
-        return [f'FROM containers-storage:{self._get_current_bootc_image()}']
+        bootc_image = self._get_current_bootc_image()
+
+        if bootc_image.startswith(LOCALHOST_BOOTC_IMAGE_PREFIX):
```

Comment:
Would be nice to be more explicit here and use a regex match. Locally I also make `tmt:debug` container images and would be nice to make sure it doesn't affect those.

Or alternatively we do it for all `loclahost/`. Iiuc that would also be desirable?